styles.css

```diff
@@ -84,4 +84,63 @@
     align-items: center;
     margin: auto;
     gap:65px;
-}
\ No newline at end of file
+}
+.content{
+  text-align: center;
+  max-width: 450px;
+  display:flex ;
+  flex-direction: column;
+  margin: 0 auto;
+  box-shadow: 30px 20px  rgb(250, 249, 249);
+}
+.title-content{
+  font-family:'Inter', sans-serif; 
+  font-size: 38px;
+  font-weight: 800;
+  line-height: 64px; 
+}
+.desc-content{
+  font-family:'Inter', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 32px; 
+}
+.feauters{
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  margin: 0 auto;
+
+}
+.feauter-wrapper{
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  justify-content: center;
+  text-align: center;
+  max-width: 390px;
```

Comment:
- [ ] This is supposed to be in the individual `features-elements` class
- [ ] Add a `max-width` and a `margin: 0 auto`